shared/applicability/package.yml

```diff
@@ -0,0 +1,12 @@
+name: 'cpe:/a:{arg}:{ver_cpe}'
+title: 'Package {arg} {ver_str} is installed'
```

Comment:
What if the `ver_str` isn't defined?

I also find that it leads to a problem that the CPE ends with colon, eg. `name="cpe:/a:chronytest:"`. Is that OK?

Reply:
and it also causes superflous space in the `title` element:
```
<cpe-dict:title xml:lang="en-us">Package chronytest  is installed</cpe-dict:title>
```
There are 2 spaces between `chronytest` and `is`.